core/src/main/java/org/elasticsearch/action/search/MultiSearchRequest.java

```diff
@@ -60,6 +61,25 @@ public MultiSearchRequest add(SearchRequest request) {
         return this;
     }
 
+    /**
+     * Returns the amount of search requests specified in this multi search requests are allowed to be ran in parallel.
+     */
+    public int maxParallelSearchRequests() {
```

Comment:
Nit: I think it should the name should reflect concurrency rather than parallelism.